docs/integrations/sources/mongodb-v2.md

```diff
@@ -160,10 +160,28 @@ The MongoDB source utilizes change data capture (CDC) as a reliable way to keep
 
 Airbyte utilizes [the change streams feature](https://www.mongodb.com/docs/manual/changeStreams/) of a [MongoDB replica set](https://www.mongodb.com/docs/manual/replication/) to incrementally capture inserts, updates and deletes using a replication plugin. To learn more how Airbyte implements CDC, refer to [Change Data Capture (CDC)](https://docs.airbyte.com/understanding-airbyte/cdc/).
 
+### No Schema Mode
```

Comment:
Can we change where this is laid out? The implication right now is customers need to pick between CDC and this new mode, which isn't quite true.

We should add a new section called # Schema Enforcement, or something like that, then lay out both options (strict schema + schemaless) in that section.